stories/object-list/object-list.stories.js

```diff
@@ -0,0 +1,1193 @@
+import '../../dist/list.css';
+import '../../dist/icon.css';
+import '../../dist/object-list.css';
+
+export default {
+    title: 'Components/Object List',
+    parameters: {
+        description: `
+An object list is another type of list that consists of object list items, and each item provides a quick 
+overview of an object. Object list items should only contain essential information for the user to identify what 
+objects to prioritize. Avoid long descriptive texts as the text space is limited and will truncate.
+         
+**Object list items can display the following elements:**
+        
+- Title of the object
+- Avatar
+- Object display components such as **Object Identifier**, **Object Number**, **Object Attribute**, **Object Marker** and **Object Status**
+- Introductory text indicating the origin of the object, such as _Forwarded by..._ or _On Behalf of..._
+- Icon that identifies the object
+- The first status line can contain indicator icons for locked items, favorites, or items that have been
+  flagged for follow-up
+
+##Usage
+**Use the object list if:**
+
+- You want to display a brief details to user.
+
+**Do not use the object list if:**
+
+- You want to display a detailed information to user.
```

Comment:
**Use the object list if:**

- You want to display brief details to the user.

**Do not use the object list if:**

- You want to display detailed information to the user.
- You want to trigger more than one event. Instead, consider using **Action List Item**.